spec/services/search/radius_suggestions_builder_spec.rb

```diff
@@ -16,57 +16,29 @@
 
     let(:arguments_for_algolia) { { aroundLatLng: Geocoder::DEFAULT_STUB_COORDINATES, hitsPerPage: 10, page: 1 } }
 
-    context "when there are 5 wider radii" do
-      let(:radius) { "1" }
-      let(:vacancies_1) { double("vacancies") }
-      let(:vacancies_2) { double("vacancies") }
-      let(:vacancies_3) { double("vacancies") }
-      let(:vacancies_4) { double("vacancies") }
-      let(:vacancies_5) { double("vacancies") }
-
-      before do
-        mock_algolia_search(
-          vacancies_1, 1, "maths",
-          arguments_for_algolia.merge(aroundRadius: convert_miles_to_metres(5))
-        )
-        mock_algolia_search(
-          vacancies_2, 1, "maths",
-          arguments_for_algolia.merge(aroundRadius: convert_miles_to_metres(10))
-        )
-        mock_algolia_search(
-          vacancies_3, 3, "maths",
-          arguments_for_algolia.merge(aroundRadius: convert_miles_to_metres(15))
-        )
-        mock_algolia_search(
-          vacancies_4, 4, "maths",
-          arguments_for_algolia.merge(aroundRadius: convert_miles_to_metres(20))
-        )
+    before do
+      search_hits.zip(wider_radii).each do |search_hits_count, radius_option|
         mock_algolia_search(
-          vacancies_5, 7, "maths",
-          arguments_for_algolia.merge(aroundRadius: convert_miles_to_metres(25))
+          double("vacancies"), search_hits_count, "maths",
+          arguments_for_algolia.merge(aroundRadius: convert_miles_to_metres(radius_option))
         )
       end
+    end
+
+    context "when there are 5 wider radii" do
+      let(:radius) { "1" }
+      let(:wider_radii) { Search::RadiusSuggestionsBuilder::RADIUS_OPTIONS[1..5] }
```

Comment:
Although I'd prefer if this/below did the same thing (I lean maybe towards below because then I can just see the numbers right here)